rpm/sailfish-setup.spec

```diff
@@ -73,6 +73,33 @@ if ! getent passwd sailfish-fingerprint >/dev/null ; then
     useradd -r -g sailfish-authentication -d / -s /sbin/nologin sailfish-fingerprint || :
 fi
 
+ALIEN_BASE_UID=500000
+if ! getent group appsupport-root >/dev/null ; then
+    groupadd -g $ALIEN_BASE_UID appsupport-root
+fi
+
+if ! getent group appsupport-system >/dev/null ; then
+    ALIEN_SYSTEM=$(expr $ALIEN_BASE_UID + 1000)
```

Comment:
You can just call `var=$((1+2))` instead of using expr here.

Reply:
Even with busybox sh?

Reply:
Yes it's a POSIX feature.